### ISSUE TITLE
[Bug]: uwu by @something in footer is not responsive from around 1500px to 1670px

### ISSUE THREAD

**Issue by DeveloperAmrit** [reactjs/react.dev#7387]
### Summary

On react documentation website. After clicking uwu, the text "Logo by @sawaratsuki1004" is breaking. Not looking good.
Make it inline for this range of screen width. (1500px to 1670px). Works fine out of this range

### Page

https://react.dev/learn/referencing-values-with-refs

### Details

On react documentation website. After clicking uwu, the text "Logo by @sawaratsuki1004" is breaking. Not looking good.
Make it inline for this range of screen width. (1500px to 1670px). Works fine out of this range

![reactbug](https://github.com/user-attachments/assets/19e9d1e5-5d28-4dda-b6bc-ddcdb0757fef)

Fix,
white-space: nowrap;

Add this css property to the tag.

**Comment by DeveloperAmrit** [comment]
Can you assign me this issue?

**Comment by karthik-awebon** [comment]
@gaearon Yes, this issue happens with the screen width range of (1500px to 1670px) but it looks to be minor and doesn't affect the documentation.

**Comment by DeveloperAmrit** [comment]
It affects User Experience for all laptop users.
